web/package.json

```diff
@@ -9,9 +9,15 @@
   },
   "scripts": {
     "start": "parcel",
-    "build": "parcel build"
+    "build": "parcel build",
+    "lint": "eslint src/",
+    "fix": "eslint --fix src/"
   },
   "devDependencies": {
-    "parcel": "^2.0.0"
+    "eslint-config-google": "^0.14.0",
```

Comment:
Let's remove the leading `^` for more consistent builds.

Reply:
If we do this we may want to make sure we have dependabot or something for the repo?

Reply:
Oh, thinking more I think your concern is that as a linter it is going to cause noise. I'll pin these. I still think having deps checking would be nice ;)

Reply:
Yes, we'll want dependabot or similar. Probably set at a weekly cadence as the NPM ecosystem is a busy one.